.changeset/full-houses-laugh.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/aqueduct": major
+"@fluidframework/datastore": major
+"@fluidframework/runtime-definitions": major
+"@fluidframework/test-runtime-utils": major
+---
+
+`bindToContext` API removed
+
+`bindToContext` has been removed from `IDataObjectProps`, `FluidDataStoreRuntime`, `IFluidDataStoreContext` and `MockFluidDataStoreContext`. This has been deprecated for a while and cannot be used anymore.
```

Comment:
IDataObjectProps didn't expose bindToContext, it exposed the context which broke.  So can you update the phrasing?  Maybe pull to yet another md file if that's the recommended approach?

Reply:
Removed mentioning IDataObjectProps